spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/repository/query/SpannerStatementQueryExecutor.java

```diff
@@ -315,10 +315,10 @@ public static Statement getChildrenRowsQuery(Key parentKey,
 	 */
 	private static String combineWithAnd(String cond1, String cond2) {
 		if (StringUtils.isEmpty(cond1)) {
-			return StringUtils.defaultIfEmpty(cond2, StringUtils.EMPTY);
+			return !StringUtils.isEmpty(cond2) ? cond2 : "";
```

Comment:
IIRC `isEmpty` is deprecated in favor of `hasText()`

Reply:
Hmm, I tried, but it seems like `isEmpty` checks if a string is empty/null however `hasText` does that plus check if it's all whitespace. There some tests that break from the change.

I initially changed this to get rid of the `import org.apache.commons.lang3.StringUtils`; upgrading to libraries-bom 16.2.0 seems to have lost this transitive dependency.